src/variations.cc

```diff
@@ -28,7 +28,7 @@ bool ParseVariationRegionList(const ots::Font* font, const uint8_t* data, const
   if (!fvar) {
     return OTS_FAILURE_MSG("Required fvar table is missing");
   }
-  if (axisCount != fvar->AxisCount()) {
+  if (*regionCount > 0 && axisCount != fvar->AxisCount()) {
```

Comment:
I think you should check for `*regionCount == 0` right after reading it above and return true, since basically the rest of this function will be no-op in this case.

Reply:
thanks, done.